packages/plugin-table/src/TablePickerDomObjectRenderer.ts

```diff
@@ -8,19 +8,29 @@ import { Table } from './Table';
 import { TableNode } from './TableNode';
 import { Layout } from '../../plugin-layout/src/Layout';
 import { TablePickerNode } from './TablePickerNode';
+import { RenderingEngineWorker } from '../../plugin-renderer/src/RenderingEngineCache';
 
 export class TablePickerDomObjectRenderer extends NodeRenderer<DomObject> {
     static id = DomObjectRenderingEngine.id;
     engine: DomObjectRenderingEngine;
     predicate = TablePickerNode;
 
-    async render(tablePicker: TableNode): Promise<DomObjectElement> {
-        const domObject = (await this.super.render(tablePicker)) as DomObjectElement;
+    async render(
+        tablePicker: TableNode,
+        worker: RenderingEngineWorker<DomObject>,
+    ): Promise<DomObjectElement> {
+        const domObject = (await this.super.render(tablePicker, worker)) as DomObjectElement;
         const tablePlugin = this.engine.editor.plugins.get(Table);
         const layout = this.engine.editor.plugins.get(Layout);
 
         const close = async (): Promise<void> => {
-            await layout.remove('TablePicker');
+            setTimeout(() => {
```

Comment:
Pourquoi un `setTimout` ? Ici `close` c'est déjà dans une fonction appelée asynchronously.

Reply:
oui... car il fait appel mémoire et si tu click sur un autre bouton il sont en même temps... comme quand on click sur undo

Reply:
Add a `TODO: remove this setTimeout when we have external vs internal execCommand`